ci/run_easymesh_cert.sh

```diff
@@ -80,7 +80,7 @@ main() {
     run "$TOOLS_PATH"/deploy_ipk.sh "$TARGET_DEVICE_SSH" "$PRPLMESH_IPK"
 
     info "Start running tests"
-    run "$EASYMESH_CERT_PATH"/run_test_file.sh -o "$LOG_FOLDER" -d "$TARGET_DEVICE" "$TESTS" "$VERBOSE_OPT"
+    "$EASYMESH_CERT_PATH"/run_test_file.sh -o "$LOG_FOLDER" -d "$TARGET_DEVICE" "$TESTS" "$VERBOSE_OPT"
```

Comment:
Yes it works:
```
Done
run_easymesh_cert.sh: Start running tests
The following test(s) will be run:

MAP-4.2.1 MAP-4.3.2_ETH
Applying radio info configuration
Using the following MasterTestInfo.xml: /home/tester/work/easymesh_cert/config/mastertestinfo/default.xml

Running test MAP-4.2.1 (1/2)

Time: 2020-03-27_15-30-29

Applying configuration for an agent tes
```

This was a suggestion by `shellcheck`. Originally I used `"$@"`, and this was the error:
```
Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.shellcheck(SC2124)
```

Reply:
Wow... This is an artefact from how run_test_file.sh is written.

As Raphaél observed, the tests will be passed a single argument to run_test_file.sh.

The read_test_names parses that into one line per argument. So all the explicitly specified tests are on a single line. You can see that in the output above: normally, the "The following test(s) will be run" output should print one test per line.

But then, the _run_tests function uses `wc -w` to get the number of tests, and `for t in $test_names` (unquoted!) to iterate over them. So for both cases, word splitting is done again, and it magically works.

It still is a horrible hack though.

The proper solution would be to switch to bash and user arrays.

Reply:
Lets do this in a separate PR then and enjoy this moment of bliss that something magically works :)